test/parallel/test-fs-read-offset-null.js

```diff
@@ -31,12 +40,16 @@ let filehandle = null;
 // Test for promise api
 (async () => {
   filehandle = await fsPromises.open(filepath, 'r');
-  const readObject = await filehandle.read(buf, null, buf.length);
+  const readObject = await filehandle.read(buf, { offset: null });
+  assert.strictEqual(readObject.buffer[0], 120);
+})()
+.then(common.mustCall())
+.finally(() => filehandle?.close());
```

Comment:
Not related to this PR, but we probably want the `.close()` call to succeed.
```suggestion
.finally(() => filehandle?.close())
.then(common.mustCall());
```

Reply:
Corrected. Nice catch!